package.json

```diff
@@ -120,6 +120,70 @@
     "node-fetch": "^2.6.1",
     "recursive-watch": "^1.1.4"
   },
+  "peerDependencies": {
```

Comment:
I had recently done this analysis because I didn't realize that there was an open PR already out here.  Comparing this to what I had come up with there's a few differences:

1. I had found a few additional dependencies that fell into this category.
2. My approach was to avoid pinning to a specific patch release.  So for versions >= 1 I used `^{major}.0.0` and for versions < 1 I used `^0.{minor}.0`.

That led to a dependency list that looked like:

```json
  "peerDependencies": {
    "amqplib": "^0.7.0",
    "avsc": "^5.0.0",
    "bunyan": "^1.0.0",
    "cbor-x": "^0.6.0",
    "dd-trace": "^0.31.0",
    "debug": "^4.0.0",
    "etcd3": "^1.0.0",
    "ioredis": "^4.0.0",
    "jaeger-client": "^3.0.0",
    "kafka-node": "^5.0.0",
    "log4js": "^6.0.0",
    "mqtt": "^4.0.0",
    "msgpack5": "^5.0.0",
    "nats": "^1.0.0",
    "node-nats-streaming": "^0.3.0",
    "notepack.io": "2.0.0",
    "pino": "^6.0.0",
    "protobufjs": "^6.0.0",
    "redlock": "^4.0.0",
    "rhea-promise": "^1.0.0",
    "thrift": "^0.14.0",
    "winston": "^3.0.0"
  },
```

Could you update this list?

Reply:
If you have a list of extensions+versions that you like and you have a different formatting that you like, you can just fix this yourself. I don't see the point in having me doing this extra work, if you've already done it.